out/blocks/wd_admin_js_resources.tpl

```diff
@@ -0,0 +1 @@
+[{oxscript include="js/libs/jquery.min.js"}]
```

Comment:
Do we really need jQuery for this? It's only used for `$.post`, which can easily be done in vanilla Javascript as well.

Reply:
This was used mainly out of convenience, since OXID includes jQuery on every page call in the admin area already anyway and did not involve adding any new dependencies.